modules/reference/pages/systems/custom-system-info.adoc

```diff
@@ -1,6 +1,8 @@
 [[ref.webui.systems.cust-info]]
 = Custom System Information
 
+// Check modules/client-configuration/pages/custom-info.adoc --LKB 2020-07-29
+
 {productname} customers may include completely customizable information about their systems.
```

Comment:
```suggestion
{productname} users may include completely customizable information about their systems.
```
We also use this in the context of Uyuni.

Reply:
I haven't edited this section yet. Just added the comment at the top so I know where to go when I get to this page.